src/api/subscriptions.js

```diff
@@ -231,6 +231,14 @@ export let LoginRequest;
  */
 export let SaveSubscriptionRequest;
 
+/**
+ * Callback for retrieving subscription request
+ *
+ * @callback SaveSubscriptionRequestCallback
+ * @return {!Promise<SaveSbuscriptionRequest> | !SaveSubscriptionRequest} request
+ */
+export let SaveSubscriptionRequestCallback;
```

Comment:
`saveSubscription()` above still needs to be updated to use this type.

Reply:
done